client/src/cmdscript.c

```diff
@@ -424,6 +429,17 @@ static int CmdScriptRun(const char *Cmd) {
 
         // clean up
         PyConfig_Clear(&py_conf);
+#else
+        wchar_t *py_args[argc + 1];
+        py_args[0] = Py_DecodeLocale(filename, NULL);
+        for (int i = 0; i < argc; i++) {
+            py_args[i + 1] = Py_DecodeLocale(argv[i], NULL);
```

Comment:
Just noticed that the index might be off-by-one.
```c
for (int i = 0; i < argc + 1; i++) {
  py_args[i] = Py_DecodeLocale(argv[i], NULL);
```
This should be enough and will already handle filename (i.e. line 434 can ve removed)

Reply:
I've noticed it also that this code puts file name into args[0] and shifts all parameter by one. This was the way the code was working into commit #38cb327991501325aab6b24310e5428dc0369ab0. TBH I didn't dig more into this.

Reply:
@jmichelp tested with parity.py and you have right! That code was wrong!

[usb] pm3 --> script run parity.py 10 1234
[+] executing python /data/home/nitro/Downloads/NFC-RFID/iceman_builds/proxmark3_08/client/pyscripts/parity.py
[+] args '10 1234'
Traceback (most recent call last):
  File "parity.py", line 75, in <module>
    main()
  File "parity.py", line 61, in main
    numtype=  int(sys.argv[1], 10)
ValueError: invalid literal for int() with base 10: 'parity.py'

Fixing it as suggested works well

[usb] pm3 --> script run parity.py 10 1234
[+] executing python /data/home/nitro/Downloads/NFC-RFID/iceman_builds/proxmark3_08/client/pyscripts/parity.py
[+] args '10 1234'
numtype: 10
num: 1234 0x4D2
Even parity

[+] finished parity.py